src/views/TabView/TabBarBottom.js

```diff
@@ -179,14 +206,19 @@ class TabBarBottom extends React.PureComponent<Props> {
 
     const tabBarStyle = [
       styles.tabBar,
-      isLandscape && useHorizontalTabs
-        ? styles.tabBarLandscape
-        : styles.tabBarPortrait,
+      this.shouldUseHorizontalTabs() && !isPad
+        ? styles.tabBarCompact
+        : styles.tabBarRegular,
       style,
     ];
 
     return (
-      <Animated.View style={animateStyle}>
+      <Animated.View
+        style={animateStyle}
+        onLayout={evt => {
+          this.setState({ viewWidth: evt.nativeEvent.layout.width });
```

Comment:
`layout` is already passed as a prop to `TabBarBottom`

Reply:
It is? I don't see it in the Prop types.  Is it called something else?